CONTRIBUTING.md

```
@@ -256,4 +504,4 @@ Include:
 
 ---
 
-We appreciate your contributions to Lamatic AgentKit!
\ No newline at end of file
+We appreciate your contributions to Lamatic AgentKit! 🚀
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)